attestation/commandrun/commandrun.go

```diff
@@ -81,32 +91,89 @@ func New(opts ...Option) *CommandRun {
 	return cr
 }
 
-type ProcessInfo struct {
-	Program          string                          `json:"program,omitempty"`
-	ProcessID        int                             `json:"processid"`
-	ParentPID        int                             `json:"parentpid"`
-	ProgramDigest    cryptoutil.DigestSet            `json:"programdigest,omitempty"`
-	Comm             string                          `json:"comm,omitempty"`
-	Cmdline          string                          `json:"cmdline,omitempty"`
-	ExeDigest        cryptoutil.DigestSet            `json:"exedigest,omitempty"`
-	OpenedFiles      map[string]cryptoutil.DigestSet `json:"openedfiles,omitempty"`
-	Environ          string                          `json:"environ,omitempty"`
-	SpecBypassIsVuln bool                            `json:"specbypassisvuln,omitempty"`
-}
-
 type CommandRun struct {
-	Cmd       []string      `json:"cmd"`
-	Stdout    string        `json:"stdout,omitempty"`
-	Stderr    string        `json:"stderr,omitempty"`
-	ExitCode  int           `json:"exitcode"`
-	Processes []ProcessInfo `json:"processes,omitempty"`
-
+	Cmd                  []string      `json:"cmd"`
+	Stdout               string        `json:"stdout,omitempty"`
+	Stderr               string        `json:"stderr,omitempty"`
+	ExitCode             int           `json:"exitcode"`
+	Processes            []ProcessInfo `json:"processes,omitempty"`
+	Files                []FileInfo    `json:"files,omitempty"`
```

Comment:
we'll probably want this to be a `map[string]FileInfo` so we can look into it by path easily

Reply:
done